frontend/env/project_local.js

```diff
@@ -16,6 +16,10 @@ module.exports = global.Project = {
   flagsmithClientEdgeAPI: 'https://edge.api.flagsmith.com/api/v1/',
   // This is used for Sentry tracking
   maintenance: false,
+  plans: {
```

Comment:
plans: {
    scaleUp: { annual: 'scale-up-annual-v2', monthly: 'scale-up-v2' },
    startup: { annual: 'startup-annual-v2', monthly: 'startup-v2' },
  },

Reply:
@novakzaballa this is what I was getting at.